nntrainer/tensor/blas_interface.cpp

```diff
@@ -883,4 +896,16 @@ void inv_sqrt_inplace(const unsigned int N, float *X) {
 #endif
 }
 
+void swish_inplace(const unsigned int N, float *X) {
+#ifdef USE_NEON
+  nntrainer::neon::swish_inplace_neon(N, X);
+#else
+  unsigned int i = 0;
+  while (i < N) {
+    X[i] = (X[i] / (1.0f + std::exp(-1 * X[i])));
```

Comment:
I know that quite a bit of code in nntrainer uses -x instead of -1 * x.
(For example, https://github.com/nnstreamer/nntrainer/blob/faadfbccce1e6a795efd3755f2302167188c0f4d/nntrainer/layers/acti_func.h#L261)

How about changing it to -x form for consistency?